cmd/configschema/configschema/resolver.go

```
@@ -52,7 +57,85 @@ func NewDirResolver(srcRoot string, moduleName string) DirResolver {
 }
 
 // PackageDir accepts a Type and returns its package dir.
+// If package is not found locally, searches for package location through go.mod.
+// Attempts to search through an absolute path or GOPATH
 func (dr DirResolver) PackageDir(t reflect.Type) string {
 	pkg := strings.TrimPrefix(t.PkgPath(), dr.ModuleName+"/")
+	dir := localPackageDir(dr, pkg)
+	_, err := os.ReadDir(dir)
+	if err != nil {
+		if dir, err = externalPackageDir(dr, pkg); err != nil {
+			panic(fmt.Sprintf("Could not find the pkg: %s", pkg))
```

Can we return an error to the caller of `configschema.ReadFields`